admin/sql/updates/2021-04-12-new-music-services-table.sql

```diff
@@ -0,0 +1,25 @@
+BEGIN;
+
+CREATE TYPE external_auth_service AS ENUM ('spotify', 'youtube');
+
+CREATE TABLE external_auth (
```

Comment:
Did I glance over the design document and miss that this was the proposed name? :)
how about `external_service_oauth` to make it a bit more explicit about what kind of external it is?

Reply:
Makes sense. `external_service_oauth` is clearer.